scout/server/extensions.py

```diff
@@ -27,20 +33,169 @@
 from flask_mail import Mail
 mail = Mail()
 
-from loqusdb.plugins import MongoAdapter as LoqusDBMongoAdapter
 from scout.adapter.client import get_connection
 
 LOG = logging.getLogger(__name__)
 
+def execute_command(cmd):
+    """
+        Prints stdout + stderr of command in real-time while being executed
+
+    Args:
+        cmd (list): command sequence
+
+    Yields:
+        line (str): line of output from command
+    """
+    output = ''
+    try:
+        output = subprocess.check_output(
+            ' '.join(cmd),
+            shell=True
```

Comment:
I do not understand how that would work but I'll fix that if you say so!

Reply:
I dont think you really need anything from the shell in this case anyway? You just need to fork off a subproc and grab the return value. Otherwise. you could try to second guess an attacker by escaping any fun characters. C f below

![Screenshot 2019-11-19 at 15 16 38](https://user-images.githubusercontent.com/758570/69154092-aa567900-0adf-11ea-8d90-9a256a7eebaf.png)